src/domains/pay-wallet-app/api/io-payment-wallet/v1/_openapi.json.tpl

```diff
@@ -604,8 +604,9 @@
           },
           "lastUsage": {
             "type": "string",
-            "description": "Time of last usage of this wallet by the client",
-            "format": "date-time"
+            "description": "(DEPRECATED\\: use ecommerce-io GET /user/lastPaymentMethodUsed to retrieve user last used method) Time of last usage of this wallet by the client\n",
```

Comment:
why this domain also modified?

Reply:
Will `lastUsage` in `pay-wallet` domain be deprecated?

Reply:
Yes, this field is populated with the _update wallet usage_ logic that have been dismantled, no one will save that data to DB anymore since the last used payment method now is stored into the user-stats service for both guest payment methods and wallet ones.
Marked as deprecated here in order to remove it in a future version